pkgs/applications/misc/rtl-sdr/default.nix

```diff
@@ -26,7 +26,13 @@ stdenv.mkDerivation rec {
     grep -q "Requires:" "$pcfile" && { echo "Upstream has added 'Requires:' in $(basename "$pcfile"); update nix expression."; exit 1; }
     echo "Requires: libusb-1.0" >> "$pcfile"
   '';
-
+  patches = lib.optionals stdenv.isDarwin [
```

Comment:
Would it be possible/pertinent to open an upstream issue?

Reply:
Considering that the version didn't change, I assume the error is due to some Nix oddity. I can dig into it a bit more